cicd/gitops/collecting-debugging-data-for-support.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="collecting-debugging-data-for-support"]
+= Collecting debugging data for a support case
+include::_attributes/common-attributes.adoc[]
+:context: collecting-debugging-data-for-support
+
+toc::[]
+
+When you open a support case, you need to provide debugging information about your cluster to the Red Hat Support team. You can use the must-gather tool to collect diagnostic information for project-level resources, cluster-level resources, and {gitops-title} components. 
```

Comment:
```suggestion
When you open a support case, you must provide debugging information about your cluster to the Red Hat Support team. You can use the must-gather tool to collect diagnostic information for project-level resources, cluster-level resources, and {gitops-title} components. 
```
suggestion per the IBM style guide:
```
Use need to only when must is too restrictive. For example, you do not need to is not the same as you must not.
For a mandatory action, either use the imperative, as in “back up your data”, or use must, as in “you must back up your data”. However, for a conditional action, you can write “you might need to back up your data”.
```
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n